src/components/item/item.ios.scss

```diff
@@ -227,6 +230,14 @@ $item-ios-sliding-content-background:     $list-ios-background-color !default;
     background-position: right ($item-ios-padding-end - 2) center;
     background-size: 14px 14px;
```

Comment:
instead of having the selector another time, in this line you can do:
```
@include rtl() {
  @include svg-background-image($item-ios-detail-push-svg-rtl);
  background-position: left ($item-ios-padding-right - 2) center;
}
```

And it will know which selector you refer to.

Same for all other platforms.

Here is the current state of RTL, in a guide, to see how to use it, and just to gain knowledge of what was done and what will be done: https://github.com/AmitMY/ionic-site/blob/6b891dfa46534da415b3a44d0752fe9ae5619f00/content/docs/rtl-support/index.md

Note, that it also describes https://github.com/driftyco/ionic/pull/11635 and https://github.com/driftyco/ionic/pull/11649 which are still open